.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -281,7 +281,7 @@
   - |
     if [[ "${CI_COMMIT_BRANCH}" = "master" ]]; then
       echo "Skipping, Should not run on master branch."
-    elif [ 'true' = $(./Tests/scripts/check_if_branch_exist.sh -u "gitlab-ci-token" -t "${CI_JOB_TOKEN}" -h "code.pan.run" --repo "xsoar/content-test-conf" -b "${CI_COMMIT_BRANCH}") ]; then
+    elif [ 'true' = $(./Tests/scripts/check_if_branch_exist.sh -u "gitlab-ci-token" -t "${CI_JOB_TOKEN}" -h "${CI_SERVER_HOST}" --repo "${CI_PROJECT_NAMESPACE}/content-test-conf" -b "${CI_COMMIT_BRANCH}") ]; then
```

Comment:
We'll need to remember to revisit this in the future when we merge `content-test-conf`, `infra`, and whatever we extract from `content`.